connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -57,6 +57,9 @@ message SqlCommand {
 
   // (Optional) A map of parameter names to literal expressions.
   map<string, Expression.Literal> args = 2;
+
+  // (Optional) A map of parameter names to column expressions.
+  map<string, Expression.UnresolvedAttribute> columns = 3;
```

Comment:
Add columns field, to support convert column to string on the server, to solve the problem connect client without jvm.